util/s3_repository.go

```diff
@@ -52,22 +52,24 @@ func FileInfoHeader() string {
 }
 
 func (f *FileInfo) String() string {
-	return fmt.Sprintf("%-50s %-50s %-15s %-20s", f.Namespace+"/"+f.Name, f.Description, f.Version, f.Created)
+	name := strings.TrimLeft(f.Namespace+"/"+f.Name, "/")
```

Comment:
Please, add a comment explaining that you are doing this because `f.Namespace` can be empty and are thus making sure that you don't get image names like `/myimage`.